### DISCUSSION TITLE
Usage statistics [Jun 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5920] [Discord Threads]
> 🙋‍♂️ **Natali Nachteule**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Not that I am aware of. Will check with the team and get back to you if we have any mechanisms. You can request for an enhancement on Github  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1105775193240117259)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
